### ISSUE TITLE
Animations Keyframes Ignore Delay

### ISSUE THREAD

**Issue by bpearce6942** [angular/angular#14144]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Animations using keyframes are ignoring the delay time specified.

**Expected behavior**
Should delay before animating.

**Minimal reproduction of the problem with instructions**
Plunkr showing Angular animation using keyframes and straight css version of same.  Css version waits 3s then hides for 3s.  Angular version hides for 6s with no delay.
https://plnkr.co/edit/3mx7CSc4pw9cMRnrfzUc?p=preview

**What is the motivation / use case for changing the behavior?**
Worked as expected prior to version 2.4.2.

* **Angular version:** 
Version 2.4.1 worked as expected.  2.4.2+ exhibits current behavior.

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
Chrome

**Comment by Toxicable** [comment]
@bpearce6942 you have exceptions being thrown in your plunkr,, can you please reproduce this in an update to date version
https://plnkr.co/edit/tpl:AvJOMERrnz94ekVua0u5

**Comment by bpearce6942** [comment]
https://plnkr.co/edit/nS60BTAvrNhrwv7EgmMJ?p=preview

**Comment by matsko** [comment]
@bpearce6942 this is fixed in 4.0.0: https://plnkr.co/edit/iCDHnmr420iuTjhTqQl7?p=preview
